spec/latest/core/core.jsonld

```diff
@@ -505,6 +518,112 @@
       "domain": "hydra:IriTemplateMapping",
       "range": "xsd:string",
       "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:target",
+      "@type": "rdf:Property",
+      "label": "invocation target",
+      "comment": "Explicit target of the invocation of the operation.",
+      "domain": "hydra:Operation",
+      "rangeIncludes": ["hydra:IriTemplate", "hydra:Resource"],
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:returnsHeader",
+      "@type": "rdf:Property",
+      "label": "returns header",
+      "comment": "Name of the header returned by the operation.",
+      "domain": "hydra:Operation",
+      "range": "xsd:string",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:expectsHeader",
+      "@type": "rdf:Property",
+      "label": "expects header",
+      "comment": "Specification of the header expected by the operation.",
+      "domain": "hydra:Operation",
+      "rangeIncludes": ["hydra:Header", "hydra:HeaderTemplate"],
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:Template",
+      "@type": "rdfs:Class",
+      "label": "Template",
+      "comment": "Some abstract template.",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:Header",
+      "@type": "rdfs:DataType",
+      "label": "Header",
+      "comment": "Specification of the header, including name and value(s).",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:HeaderTemplate",
+      "@type": "rdfs:Class",
+      "subClassOf": "hydra:Template",
+      "label": "Header template",
+      "comment": "Specification of the header template with variables to be replaced with actual values.",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:name",

```

Comment:
hydra:name sounds very generic, as if you could name anything. Only the comment states, it is used to name a header. I would prefer hydra:headerName

Reply:
Good point. Here you go!